covid19_sfbayarea/data/san_mateo/power_bi_querier.py

```diff
@@ -0,0 +1,134 @@
+import json
+from functools import reduce
+from requests import post
+from typing import Any, Dict, List
+
+class PowerBiQuerier:
+    BASE_URI = 'https://wabi-us-gov-iowa-api.analysis.usgovcloudapi.net/public/reports/querydata?synchronous=true'
+    JSON_PATH = ['results', 0, 'result', 'data', 'dsr', 'DS', 0, 'PH', 0, 'DM0']
```

Comment:
It looks like they just replaced DM0 with DM1 in the hospitalization dashboard: 1ec5/external-data-bot@fba0e0f0e364b294f367c44cef7dece08b6f861e. I have no idea what the difference is.

Reply:
Thanks! I will keep an eye out for stuff like this - for now we don't use hospitalization data, but that could be a harbinger of more changes to come!

Reply:
@1ec5 we're getting hospitalization data from the state, see here: https://github.com/sfbrigade/data-covid19-sfbayarea/blob/master/covid19_sfbayarea/data/hospitals.py